userspace/libsinsp/grpc_channel_registry.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright (C) 2013-2019 Draios Inc dba Sysdig.
```

Comment:
Is it still Draios Inc. dba Sysdig or just Sysdig Inc.?

Reply:
I'm just using the approved format we got when we went through the oss review. I haven't heard of any other approved format.